### DISCUSSION TITLE
Various errors with a new project

### DISCUSSION THREAD

**Discussion by randomsnowflake** [excid3/jumpstart#203] [Q&A]
I'm currently shopping for a Rails SAAS starter template.

The following issues appear after starting the server with yesterdays master:

**NameError in NotificationsController#index**

http://127.0.0.1:3000/notifications

`Rails couldn't find a valid model for Notification association. Please provide the :class_name option on the association declaration. If :class_name is already provided, make sure it's an ActiveRecord::Base subclass.
`

**No welcome email when signing up**

I'm not sure if it is supposed to be like this, but I didn't receive a welcome email when signing up. 

**Avatar upload not working**

I tried to upload an avatar, but it didn't work. 

The uploaded image is not accessible via the browser. There is no error.

The image url is: 
http://127.0.0.1:3000/rails/active_storage/representations/redirect/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBZ3ciLCJleHAiOm51bGwsInB1ciI6ImJsb2JfaWQifX0=--c2321ff5c468b79397f5a3c3d8dc3dd415b77a92/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdCem9MWm05eWJXRjBTU0lJYW5CbkJqb0dSVlE2RTNKbGMybDZaVjkwYjE5bWFXeHNXd2RwQWJScEFiUT0iLCJleHAiOm51bGwsInB1ciI6InZhcmlhdGlvbiJ9fQ==--7a4561ad91b8624cc5db287899d4b02d13e99301/file.jpg

The image is not shown in the madmin active storage view (http://127.0.0.1:3000/madmin/active_storage/attachments)

**Madmin users list is broken**

http://127.0.0.1:3000/madmin/users

`Rails couldn't find a valid model for Notification association. Please provide the :class_name option on the association declaration. If :class_name is already provided, make sure it's an ActiveRecord::Base subclass.
`

**Comment by excid3** [comment]
The free Jumpstart (this repo) is not a SaaS template. The paid version on JumpstartRails.com is what you'd want to look at.

I don't know what you mean by "yesterdays master"? Are you referring to Rails main?

It sounds like you did not follow the post-install instructions to run the noticed model generator.

**Comment by stan74fr** [comment]
for "Madmin users list is broken", indeed, the command "rails generate noticed:model" shall be used once you have generated the admin dashboard (through the command "rails g madmin:install" )
